ui/app/AppLayouts/Profile/views/wallet/ManageTokensView.qml

```diff
@@ -128,10 +133,74 @@ ColumnLayout {
     }
 
     Component {
-        id: supportedTokensListPanel
-        SupportedTokenListsPanel {
-            sourcesOfTokensModel: root.sourcesOfTokensModel
-            tokensListModel: root.tokensListModel
+        id: advancedTab
+        ColumnLayout {
```

Comment:
I've just seen in the design that it's needed to add the list of tokens that are hidden bc they have low balance.  [Design](https://www.figma.com/file/eM26pyHZUeAwMLviaS1KJn/%E2%9A%99%EF%B8%8F-Wallet-Settings%3A-Manage-Tokens?type=design&node-id=305%3A142331&mode=design&t=81rOxWBKpouduNoM-1). Here a screenshot:
<img width="804" alt="Screenshot 2024-01-11 at 10 16 01" src="https://github.com/status-im/status-desktop/assets/97019400/bd77d2b3-95c0-4642-98b6-366270b3f298">

Reply:
OK, that's in the new design; I'd leave that for later

Reply:
Ok! Could you create subtask for that?

Reply:
Done here: https://github.com/status-im/status-desktop/issues/13183